components/common-go/cgroups/v2/io.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package v2
```

Comment:
I think you'll want to name it `cgroups_v2` because otherwise you'd import it as `v2.IO` which isn't very descriptive

Reply:
Why?
Shouldn't the other package import as `common-go/cgroups/v2` and calls either `v2.NewIOControllerWithMount` _or_ `v2.NewIOController`?